### DISCUSSION TITLE
Combiné optimisé NBA

### DISCUSSION THREAD

**Discussion by kreon76** [pretrehr/Sports-betting#25] [General]
Bonsoir,

Petite question pour savoir si il est normal que l'onglet "combiné optimisé" ne fonctionne pas pour le basket ?

![image](https://user-images.githubusercontent.com/78261069/108608689-e5bdb280-73c8-11eb-97d3-f31f17d575e8.png)

Peut-être est ce seulement pour le foot ?

Merci par avance

**Comment by pretrehr** [comment]
C'est corrigé, merci !

Non pas de problème on peut utiliser cette fonctionnalité pour tous les sports. 

Juste pour indication, la fonctionnalité est surtout utile sur PokerStars lorsqu'ils proposent leur "Grosse cote". Je n'ai encore jamais trouvé de surebet sur les combinés boostés des autres bookmakers.

![image](https://user-images.githubusercontent.com/43851883/108611205-88cbf780-73dc-11eb-8f40-597394eeba1b.png)


Excellente soirée

**Reply by kreon76** [reply]
Génial Merci ! 
Hier Zebet a sorti une CB NBA a 4 matchs et je voulais voir si on était loin du surebet.
Grace à la mise a jour je pourrais vérifier la prochaine fois merci encore pour le taf
